spec/verification_response_spec.rb

```diff
@@ -101,12 +113,8 @@ def replace_expires_date(hash, time)
 
     context 'for a receipt with expiration date in the future' do
       let(:verify) do
-        response["receipt"]["in_app"].each do |in_app|
```

Comment:
@samjohn why should we delete this part for `in_app`?

Reply:
oh, I did this because in this test *every* receipt has an expiration date in the future so the order doesn't matter. this way you just make the first one in the future. I think your solution was more clear though.